src/main/java/com/google/sps/data/Listing.java

```diff
@@ -1,32 +1,30 @@
 package com.google.sps.data;
+import com.google.maps.model.Photo;
 
 /** Listing object with the specified attributes returned from the places API **/
 public class Listing {
     private String name;
     private MapLocation mapLocation;
     private double rating;
-    private String image;
-    private String url;
```

Comment:
I assume we're just leaving url to another version since it will take more work to get?

Reply:
Yeah, I think that's the plan for now